controllers/forum-controllers.js

```diff
@@ -1,4 +1,4 @@
-const { selectCategories } = require("../models/models.js");
+const { selectCategories, selectReviewById } = require("../models/models.js");
```

Comment:
I would receommend separating these controllers out into sections such as categories and reviews to make things clearer